spec/index.bs

```diff
@@ -3348,14 +3360,16 @@ The {{GPUBufferUsage}} flags determine how a {{GPUBuffer}} may be used after its
         The buffer can be mapped for reading. (Example: calling {{GPUBuffer/mapAsync()}} with
         {{GPUMapMode/READ|GPUMapMode.READ}})
 
-        May only be combined with {{GPUBufferUsage/COPY_DST}}.
+        May only be combined with {{GPUBufferUsage/COPY_DST}} when
+        {{GPUFeatureName/buffer-map-extended-usages}} is not enabled.
```

Comment:
This phrasing is hard to follow. Suggest:

> Unless {{GPUFeatureName/buffer-map-extended-usages}} is enabled, this may only be combined with {{GPUBufferUsage/COPY_DST}}.